watchdog-agent/src/test/java/AgentTest.java

```diff
@@ -93,8 +95,7 @@ void sorald_0_8_5_shouldExitWith_1() throws IOException, InterruptedException {
 
         Process p = pb.start();
         int exitCode = p.waitFor();
-
-        assertThat(exitCode).isEqualTo(0);
+        assertThat(exitCode).isEqualTo(1);
```

Comment:
`org/xml/sax/SAXException` seems like an internal class. We can investigate why this is not detected by the JDK indexer.